Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -597,6 +597,7 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
         if let speedLimitView = speedLimitView {
             speedLimitView.signStandard = routeProgress.currentLegProgress.currentStep.speedLimitSignStandard
             speedLimitView.speedLimit = routeProgress.currentLegProgress.currentSpeedLimit
+            speedLimitView.currentSpeed = max(location.speed, 0.0)
```

Comment:
If we can simply get the speed out of the location like this, it doesn’t seem like we actually need `speedKey` in the notification at all.

Reply:
Thanks for the suggestion, I just removed this key.